.github/workflows/build.yml

```diff
@@ -22,11 +22,11 @@ jobs:
           sudo add-apt-repository ppa:alexlarsson/glib260
           sudo add-apt-repository 'deb https://download.mono-project.com/repo/ubuntu stable-bionic main' # Needed for updates to work
           sudo apt-get update
-          sudo apt-get install -y libglib2.0 gettext dbus gtk-doc-tools meson libgirepository1.0-dev libgtk-3-dev
+          sudo apt-get install -y libglib2.0 gettext dbus meson libgirepository1.0-dev libgtk-3-dev
       - name: Check out libportal
         uses: actions/checkout@v1
       - name: Configure libportal
-        run: meson setup --prefix=/usr _build -Dbackends=gtk3
+        run: meson setup --prefix=/usr _build -Dbackends=gtk3 -Dgtk_doc=false
```

Comment:
Shouldn't the `gtk_doc` option be removed?

Reply:
Maybe rename it to `doc`but for the sake of keeping already existing scripts working I think is better to leave it. Note that this is also done on other places like libadwaita.

Reply:
libportal is very early in its life, go ahead and rename it now rather than later.

Reply:
Done, left a comment bellow.